### ISSUE TITLE
Request for Agentic RL training details (thank you for open-sourcing!)

### ISSUE THREAD

**Issue by zkangning** [Snowflake-Labs/agent-world-model#4]
I’m trying to follow/reproduce the Agentic RL part, but some training details are not fully clear from the current docs. 

**Would you consider open-sourcing the training code/details for the Agentic RL part to make it easier for others to reproduce and build on your results?**

Thanks again for the great open-source release!

**Comment by zkangning** [comment]
@Raibows Thank you for your response. _I am likewise confused about the BFCL v3 results reported for multiple models in the paper._ According to the official results as well as my own experiments, the Qwen3-8B Base Model achieves an Overall metric of approximately **65%** on BFCL v3, whereas the paper reports **53.83%**.

 I also downloaded AWM-8B and evaluated it using the official Qwen3Handler, obtaining an Overall Score of **65.61%**.

Therefore, I would like to ask **how the BFCL v3 metrics reported in the paper are measured**.  If the official evaluation code is used, the results for the Base Models Qwen3-4B, Qwen3-8B, and Qwen3-14B should be close to **63%, 66%, 68%**, respectively. 

<img width="376" height="41" alt="Image" src="https://github.com/user-attachments/assets/ade80249-7b88-4e21-9e9e-f2a3a850ffbd" />

<img width="379" height="36" alt="Image" src="https://github.com/user-attachments/assets/5557adea-9761-4707-800a-eb7237784d8c" />

<img width="375" height="44" alt="Image" src="https://github.com/user-attachments/assets/0b3ac3e1-7077-4603-a416-52f8a5857509" />

Sincerely look forward to your reply.  If possible, **could you make the evaluation code publicly available** to help us align the metrics reported in the paper.

**Comment by Raibows** [comment]
Hi @zkangning 

I have just released the MCP unified evaluation harness for BFCLv3, tau2bench, and MCP-Universe. I'll try to merge it into the main repo.

If you want to have a try first, please checkout this #11 PR.
